Formula/samba.rb

```diff
@@ -66,13 +66,19 @@ def install
       # macOS has its own SMB daemon as /usr/sbin/smbd, so rename our smbd to samba-dot-org-smbd to avoid conflict.
       # samba-dot-org-smbd is used by qemu.rb .
       mv "#{sbin}/smbd", "#{sbin}/samba-dot-org-smbd"
+      # Rename mdfind and profiles as well to avoid conflicting with /usr/bin/{mdfind,profiles}
+      mv "#{bin}/mdfind", "#{bin}/samba-dot-org-mdfind"
+      mv "#{bin}/profiles", "#{bin}/samba-dot-org-profiles"
     end
   end
 
   def caveats
     on_macos do
       <<~EOS
-        macOS has its own SMB daemon as /usr/sbin/smbd, so Samba version of smbd is installed as "#{HOMEBREW_PREFIX}/sbin/samba-dot-org-smbd"
+        To avoid conflicting with macOS system binaries, some files were installed with non-standard name:
+        - smbd: "#{HOMEBREW_PREFIX}/sbin/samba-dot-org-smbd"
+        - mdfind: "#{HOMEBREW_PREFIX}/bin/samba-dot-org-mdfind"
+        - profiles: "#{HOMEBREW_PREFIX}/bin/samba-dot-org-profiles"
```

Comment:
```suggestion
        - smbd:     #{HOMEBREW_PREFIX}/sbin/samba-dot-org-smbd
        - mdfind:   #{HOMEBREW_PREFIX}/bin/samba-dot-org-mdfind
        - profiles: #{HOMEBREW_PREFIX}/bin/samba-dot-org-profiles
```
I think we can do away with the `"`s and align this for readability.

Reply:
thanks, updated